index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+     <!-- I copied this line of code  -->
+    <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="" id="wrap.container-fluid"></div>
+    <div class="row" id="header.col-xs-12.col-sm-12.col-md-12.col-lg-12"></div>
+    <h3 id="title">GifTastic</h3>
+    <div class="row" id="buttonArea.col-xs-12.col-sm-12.col-md-12.col-lg-12"></div>
+    <div class="row"><div class="col-xs-12.col-sm-12.col-md-12col-lg-12"></div></div>
+    <p>Click a button above to load gifs. Or choose your own topic!</p>
+    <form>
+        <label for="topic-input"> Add a Gif topic </label><br>
+        <input id="topic-input" type="text" placeholder="type something">
+        <button class="btn.submit" type="submit", data="submitButton">Submit</button>
+        <p class="instructions">Click the gif to pause and play</p>
+    </form>
+    <div class="row" id="gifArea.col-xs-12.col-sm-12.col-md-12.col-lg-12"></div>
+    <script src="javascript.js"></script>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+        integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
```

Comment:
The jQuery CDN right now is not working and so the entire app is not functional. I would try using this CDN and seeing if that corrects the issue. <script
  src="https://code.jquery.com/jquery-3.4.1.js"
  integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU="
  crossorigin="anonymous"></script>